crates/types/src/camera_matrix.rs

```diff
@@ -34,14 +40,16 @@ impl CameraMatrices {
 )]
 #[abs_diff_eq(epsilon = "f32")]
 pub struct CameraMatrix {
-    pub camera_to_head: Isometry3<f32>,
-    pub camera_to_ground: Isometry3<f32>,
-    pub ground_to_camera: Isometry3<f32>,
-    pub camera_to_robot: Isometry3<f32>,
-    pub robot_to_camera: Isometry3<f32>,
-    pub focal_length: Vector2<f32>,
-    pub optical_center: Point2<f32>,
-    pub field_of_view: Vector2<f32>,
+    pub camera_to_head: Isometry3<Camera, Head>,
+    pub camera_to_ground: Isometry3<Camera, Ground>,
+    pub ground_to_camera: Isometry3<Ground, Camera>,
+    pub camera_to_robot: Isometry3<Camera, Robot>,
+    pub robot_to_camera: Isometry3<Robot, Camera>,
+    // TODO: I don't know enough about this to say which
+    // (if any) coordinate system these belong to
```

Comment:
todo

Reply:
they definitely do, maybe we take this opportunity and discuss with ole and his latest insights?

Reply:
Removed comment, lets do this later